cat_handler.go

```diff
@@ -1,8 +1,8 @@
 package main
 
 import (
-	"github.com/go-telegram-bot-api/telegram-bot-api"
 	"github.com/zhulik/margelet"
+	"gopkg.in/telegram-bot-api.v2"
```

Comment:
Had another branch with an updated version on it, didn't want to send a PR until after that got merged. Opened one now though.